hist/hist/src/TH2.cxx

```diff
@@ -152,7 +207,7 @@ TH2::TH2(const char *name,const char *title,Int_t nbinsx,const Float_t *xbins
 
 ////////////////////////////////////////////////////////////////////////////////
 /// Copy constructor.
-/// The list of functions is not copied. (Use Clone if needed)
+/// The list of functions is not copied. (Use Clone() if needed)
```

Comment:
Private copy constructor.  One should use the copy constructor of the derived classes (e.g. TH1D, TH1F).

I would add the comment about the list of functions in TH1::Copy() and in the copy constructor of TH1D,TH1F link TH1::Copy()